Cargo.toml

```diff
@@ -63,7 +63,7 @@ matchit = "0.7.3"
 p256 = { version = "0.13.2", features = ["ecdsa-core", "ecdsa", "pem"] }
 paste = "1.0.15"
 prio_draft09 = { package = "prio", version = "0.16.7" }
-prio = { git = "https://github.com/divviup/libprio-rs.git", rev = "c50bb9a47b396ad6a08a3fec36b98bcc2d9217a1" }
+prio = { git = "https://github.com/divviup/libprio-rs.git", rev = "7e479222152cb16abc2745bf651ad93a88f8088e" }
 prometheus = "0.13.4"
```

Comment:
Before merging this PR we need to make a change upstream: https://github.com/divviup/libprio-rs/pull/1209